setup-production.adoc

```diff
@@ -31,217 +31,172 @@ $ cd taiga-docker/
+.**Queue manager settings**
+These variables are used to leave messages in the rabbitmq services.
 
-**Service: taiga-back**
 [source, bash]
 ----
-SESSION_COOKIE_SECURE: "False"
-CSRF_COOKIE_SECURE: "False"
+RABBITMQ_USER=taiga  # user to connect to RabbitMQ
+RABBITMQ_PASS=taiga  # RabbitMQ user's password
+RABBITMQ_VHOST=taiga  # RabbitMQ container name
+RABBITMQ_ERLANG_COOKIE=secret-erlang-cookie  # unique value shared by any connected instance of RabbitMQ
 ----
 
-More info about those variables can be found link:https://docs.djangoproject.com/en/3.1/ref/settings/#csrf-cookie-secure[here].
 
-.Email Settings
+.**Attachments settings**
 
-By default, email is configured with the *console* backend, which means that the emails will be shown in the stdout. If you have an smtp service, uncomment the "Email settings" section in `docker-compose.yml` and configure those environment variables:
+You can configure how long the attachments will be accessible by changing the token expiration timer. After that amount of seconds the token will expire.
 
-**Service: taiga-back**
 [source, bash]
 ----
-EMAIL_BACKEND: "django.core.mail.backends.smtp.EmailBackend"
-DEFAULT_FROM_EMAIL: "no-reply@mycompany.com"
-EMAIL_HOST: "smtp.host.mycompany.com"
-EMAIL_PORT: 587
-EMAIL_HOST_USER: "user"
-EMAIL_HOST_PASSWORD: "password"
-EMAIL_USE_TLS: "True"
-EMAIL_USE_SSL: "True"
+ATTACHMENTS_MAX_AGE=360  # token expiration date (in seconds)
 ----
 
-Uncomment `EMAIL_BACKEND` variable, but do not modify unless you know what you're doing.
 
-.Telemetry Settings
+.**Telemetry settings**
 
 Telemetry anonymous data is collected in order to learn about the use of Taiga and improve the platform based on real scenarios.
 
-**Service: taiga-back**
 [source, bash]
 ----
-ENABLE_TELEMETRY: "True"
+ENABLE_TELEMETRY=True
 ----
 
-You can opt out by setting this variable to "False". By default is "True".
+You can opt out by setting this variable to False. By default, it's True.
 
-.Rabbit settings
 
-These variables are used to leave messages in the rabbitmq services. These variables should be the same as in `taiga-back`, `taiga-async`, `taiga-events`, `taiga-async-rabbitmq` and `taiga-events-rabbitmq`.
+=== Additional customization
 
-**Service: taiga-back**
-[source, bash]
-----
-RABBITMQ_USER: taiga
-RABBITMQ_PASS: taiga
-----
+All these customization options are by default disabled and require you to edit `docker-compose.yml` and `docker-compose-inits.yml`.
 
-Two other variables `EVENTS_PUSH_BACKEND_URL` and `CELERY_BROKER_URL` can also be used to set the events push backend URL and celery broker URL.
+You should add the corresponding environment variables in the proper services with a valid value in order to enable them. Have in mind that some variables are in both files, and you need to edit both.
```

Comment:
which customizacion variables are used in both files?

Reply:
It applies to any 'environment variables` changed in the `"Additional customization".